www/templates/intro/saveTokenFile.html

```diff
@@ -5,7 +5,7 @@ <h2 ng-i18next>{{'login.make-sure-save-your-opcode'}}</h2>
 <div class="consent-text">
     <h3 ng-i18next style="text-align: center">{{'login.cannot-retrieve'}}</h3>
 
-    <qr-code ng-if="currentToken" value="currentToken" class="fill-container"
+    <qr-code ng-if="currentToken" value="qrToken" class="fill-container"
```

Comment:
good catch! I guess I was too focused on creating a reasonable saveTokenFile and not checking the token format :)